### PR TITLE
add Mint Sepolia Testnet

### PR DESCRIPTION
**Pull request by zhwindy**
### Description

`Add Mint Sepolia Testnet`

#### Type: <New | Updated> <Network | Warp Route>

`New Network`

#### Network(s): <Network Name(s)>

`Mint Sepolia Testnet`

### Testing

### CODE REVIEW

File: chains/mintsepoliatest/logo.svg
Comment (jmrossy) [inline chains/mintsepoliatest/logo.svg:1]: Due to where these get used, these logos should be just icons please

File: chains/mintsepoliatest/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=../schema.json
+name: mintsepoliatest
```
Comment (jmrossy) [inline chains/mintsepoliatest/metadata.yaml:2]: please set isTestnet to true

File: chains/mintsepoliatest/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+domainId: 1687
+protocol: ethereum
+rpcUrls:
+  - http: https://sepolia-testnet-rpc.mintchain.io
+nativeToken:
+  symbol: ETH
+  name: Ether
+  decimals: 18
+blockExplorers:
+  - apiUrl: https://sepolia-testnet-explorer.mintchain.io/api-docs
```
Comment (jmrossy) [inline chains/mintsepoliatest/metadata.yaml:14]: I think `https://sepolia-testnet-explorer.mintchain.io/api/v2/` is what you're looking for here

**Review by jmrossy — COMMENTED** [review]
Thanks for the contribution! Just a few minor comments and please also see the CI errors

File: chains/mintsepoliatest/logo.svg
Comment (zhwindy) [reply chains/mintsepoliatest/logo.svg:1]: ok, already change it to icon

File: chains/mintsepoliatest/metadata.yaml (hunk 1 above)
Comment (zhwindy) [reply chains/mintsepoliatest/metadata.yaml]: where to set the flag isTestnet?

File: chains/mintsepoliatest/metadata.yaml (hunk 2 above)
Comment (zhwindy) [reply chains/mintsepoliatest/metadata.yaml]: yep, fixed

File: chains/mintsepoliatest/metadata.yaml (hunk 1 above)
Comment (zhwindy) [reply chains/mintsepoliatest/metadata.yaml]: already set

File: chains/mintsepoliatest/metadata.yaml
Comment (jmrossy) [inline chains/mintsepoliatest/metadata.yaml:1]: Sorry I forgot to mention, we now require chain metadata to include a `deployer` field. Would you mind adding info like this: https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/chains/alfajores/metadata.yaml#L16

**Review by jmrossy — APPROVED** [review]

**Pull request merged**